ena-submission/src/ena_deposition/create_assembly.py

```diff
@@ -89,22 +92,30 @@ def get_segment_order(unaligned_sequences: dict[str, str]) -> list[str]:
     return sorted(segment_order)
 
 
-def get_address(config: Config, entry: dict[str, str]) -> str:
+def get_address(config: Config, entry: dict[str, Any]) -> str:
     address_string = entry["center_name"]
     if config.is_broker:
         try:
-            group_info = get_group_info(config, entry["metadata"]["groupId"])[0]["group"]
-            address = group_info["address"]
-            address_list = [
-                entry["center_name"],  # corresponds to Loculus' "Institution" group field
-                address.get("city"),
-                address.get("state"),
-                address.get("country"),
+            group_info = call_loculus.get_group_info(config, entry["metadata"]["groupId"])[0][
+                "group"
             ]
-            address_string = ", ".join([x for x in address_list if x is not None])
-            logger.debug("Created address from group_info")
         except Exception as e:
-            logger.error(f"Was unable to create address, setting address to center_name due to {e}")
+            logger.error(
+                f"Failed to fetch group info for groupId={entry['metadata']['groupId']}\n"
```

Comment:
maybe it would be better to log `failed to parse group info for groupId=...` as you already log request errors inside the `get_group_info` function